mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -813,8 +817,8 @@ msgstr "Tumbleworm"
 
 msgid "tumbleworm_description"
 msgstr ""
-"Lorsqu'ils ont dépassé leurs œufs en nid d'abeille, ils tombent au sol face "
-"en premier."
+"Lorsqu'ils sont plus grands que leurs œufs, ils tombent vers le sol, la tête"
```

Comment:
I would remove the coma after sol.

"[...] ils tombent vers le sol la tête la première."